src/server/grpc/util.rs

```diff
@@ -49,3 +52,21 @@ pub async fn connect_base_node(
 
     Ok(client)
 }
+
+pub fn convert_coinbase_extra(tribe: Tribe, custom_coinbase_extra: String) -> Result<Vec<u8>, TryFromIntError> {
+    let type_length_value_marker = 0xFFu8;
+    let tribe_type_marker = 0x02u8;
+    let custom_message_type_marker = 0x01u8;
+
+    let mut current_tribe = tribe.as_string().into_bytes();
+    let current_tribe_len = u8::try_from(current_tribe.len())?;
+    let mut result = vec![type_length_value_marker, tribe_type_marker, current_tribe_len];
+    result.append(&mut current_tribe);
+
+    let mut custom_coinbase_extra_bytes = custom_coinbase_extra.into_bytes();
+    let custom_coinbase_extra_len = u8::try_from(custom_coinbase_extra_bytes.len())?;
```

Comment:
you should truncate this message to less than 255 bytes

Reply:
okay let me do it, but there is a chance that it breaks a character in the end of custom coinbase extra message (when a char is stored not only on 1 byte), so what about truncate the `String` instead of the resulting `bytes` to only `254`?

Reply:
okay I leave the impl as it is as we talked about that, so now if we exceed 255 chars it will error anyway, so closing.